doc/Learn/Navigation/Advanced/HowTo-UseTabBar.md

```diff
@@ -16,6 +16,12 @@ var builder = this.CreateBuilder(args)
 
 ## Step-by-steps
 
+> [!IMPORTANT]
+> This guide assumes you used the template wizard or `dotnet new unoapp` to create your solution. If not, it is recommended that you follow the [Creating an application with Uno.Extensions article](xref:Uno.Extensions.HowToGettingStarted) for creating an application from the template.
+
+> [!NOTE]
+> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however it uses the `Mvvm` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Comment:
```suggestion
> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however, it uses the `MVVM` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Reply:
See comment here https://github.com/unoplatform/uno.extensions/pull/2314/files#r1622685604

Reply:
Resolving as it's the same as https://github.com/unoplatform/uno.extensions/pull/2314#discussion_r1622684275